hooks/pre-commit

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
tbh we should probably work towards making sentry's pre-commit hook reusable (since the VIRTUAL_ENV check here is pretty useful, and its copypasta)